test/unit/app/models/contact_test.rb

```diff
@@ -292,4 +292,21 @@ class ContactTest < ActiveSupport::TestCase
       office.contact.destroy!
     end
   end
+
+  test "when associated with a WorldwideOrganisationPage should be valid when translated into a language that the worldwide organisation has" do
```

Comment:
`when associated with a WorldwideOffice should be valid when translated into a language that the worldwide office's associated organisation has`?

Reply:
Thanks.  I copy and pasted this from the other commit, but forgot to change the model name! 🤦‍♂️